src/app/launcher/create-app/targetenvironment-createapp-step/target-environment-createapp-step.component.ts

```diff
@@ -93,6 +93,7 @@ export class TargetEnvironmentCreateappStepComponent extends LauncherStep implem
     if (selection !== undefined) {
       this.launcherComponent.summary.targetEnvironment = selection.targetEnvironment;
     }
+    this.launcherComponent.summary.cluster = selection.cluster;
```

Comment:
this would be better to have under` if (selection !== undefined)` check, this is breaking OSIO. ps review the PR https://github.com/fabric8-launcher/ngx-launcher/pull/141